Makefile

```diff
@@ -6,15 +6,17 @@ all:
 install:
 	mkdir -p $(DESTDIR)$(PREFIX)/bin
 	mkdir -p $(DESTDIR)$(PREFIX)/share/man/man1
-	mkdir -p $(DESTDIR)/etc/neofetch
+	mkdir -p $(DESTDIR)$(PREFIX)/etc/neofetch
```

Comment:
Yes, but @rekado said explicitly that they need to change `$DESTDIR` to `$PREFIX`. That's what I find weird.